pkg/trustedresources/verify.go

```diff
@@ -45,8 +45,12 @@ const (
 // Skip the verification when no policies are found and trusted-resources-verification-no-match-policy is set to ignore or warn
 // Return an error when no policies are found and trusted-resources-verification-no-match-policy is set to fail,
 // or the resource fails to pass matched enforce verification policy
-// refSourceURI is from RefSource.URI, which will be used to match policy patterns. k8s is used to fetch secret from cluster
-func VerifyTask(ctx context.Context, taskObj *v1beta1.Task, k8s kubernetes.Interface, refSourceURI string, verificationpolicies []*v1alpha1.VerificationPolicy) error {
+// refSource is used to match policy patterns by using refSource.URI. k8s is used to fetch secret from cluster
```

Comment:
this goes into implementation details which I prefer to avoid, maybe focus on what refsource represents and if there are any special requirements on refsource and k8s?